Lib/glyphsLib/builder/features.py

```diff
@@ -89,6 +89,13 @@ def _to_ufo_features(font, ufo=None, generate_GDEF=False, skip_export_glyphs=Non
         if feature.notes:
             lines.append("# notes:")
             lines.extend("# " + line for line in feature.notes.splitlines())
+            feature_name = re.search("^Name: (.+)", feature.notes, flags=re.MULTILINE)
+            if feature_name and feature_name.groups():
```

Comment:
if the returned match object is not None, I think it must contain some non empty groups so the second test `feature_name.groups()` should be redundant